suites/nautilus/upgrades/tier-2_upgrade_with-tags.yaml

```diff
@@ -0,0 +1,597 @@
+              rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+                testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: Rolling update 4.x with tags osds
+    abort-on-fail: True
+
+- test:
+    name: check-ceph-health
+    module: exec.py
+    config:
+      cmd: ceph -s
+      sudo: True
+    desc: Check for ceph health debug info
+
+- test:
+    name: check-mon-dump
+    module: exec.py
+    config:
+      cmd: ceph mon dump
+      sudo: True
+    desc: Check for ceph mon dump debug info
+
+- test:
+    name: check-osd-dump
+    module: exec.py
+    config:
+      cmd: ceph osd dump
+      sudo: True
+    desc: Check for ceph osd dump debug info
+
+- test:
+    name: Upgrade containerized ceph to 4.x latest
+    polarion-id: CEPH-83574481
+    module: test_ansible_upgrade.py
+    config:
+      build: '4.x'
+      ansi_cli_args:
+        tags: "rgws"
+      ansi_config:
+        ceph_origin: distro
+        ceph_stable_release: nautilus
+        ceph_repository: rhcs
+        ceph_rhcs_version: 4
+        osd_scenario: collocated
+        osd_auto_discovery: False
+        ceph_stable: True
+        ceph_stable_rh_storage: True
+        fetch_directory: ~/fetch
+        copy_admin_key: true
+        containerized_deployment: False
+        upgrade_ceph_packages: True
+        dashboard_enabled: True
+        dashboard_admin_user: admin
+        dashboard_admin_password: p@ssw0rd
+        grafana_admin_user: admin
+        grafana_admin_password: p@ssw0rd
+        node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
+        grafana_container_image: registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4
+        prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:v4.6
+        alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6
+        ceph_conf_overrides:
+            global:
+              osd_pool_default_pg_num: 64
+              osd_default_pool_size: 2
+              osd_pool_default_pgp_num: 64
+              mon_max_pg_per_osd: 1024
+            mon:
+              mon_allow_pool_delete: true
+            client:
+              rgw crypt require ssl: false
+              rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+                testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: Rolling update 4.x with tag rgws
+    abort-on-fail: True
+
+- test:
+    name: check-ceph-health
+    module: exec.py
+    config:
+      cmd: ceph -s
+      sudo: True
+    desc: Check for ceph health debug info
+
+- test:
+    name: check-mon-dump
+    module: exec.py
+    config:
+      cmd: ceph mon dump
+      sudo: True
+    desc: Check for ceph mon dump debug info
+
+- test:
+    name: check-osd-dump
+    module: exec.py
+    config:
+      cmd: ceph osd dump
+      sudo: True
+    desc: Check for ceph osd dump debug info
```

Comment:
please have a look if we could club these commands and have it in one test case.